frontend/public/module/auth.js

```diff
@@ -55,7 +56,6 @@ export const authSvc = {
   },
 
   login: () => {
-    setNext(window.location.pathname);
```

Comment:
Will this break direct linking to a page if you're not already logged in?

Reply:
@spadgett so I've tried to navigate to nested page in the console, copy the URL and put it into incognito tag where I've logged in as the same user and I was redirected to the nested page.

Reply:
This PR works because any 401 will trigger the logout code, which sets next.